exonum-java-binding-core/rust/generate_licenses.sh

```diff
@@ -1,7 +1,17 @@
 #!/usr/bin/env bash
 # Gather licenses for every Rust dependency and print it in the file in a readable form.
 
-cargo license -h || cargo install cargo-license
+cargo license -h &> /dev/null || cargo install cargo-license
 
-cargo license -da | sed 's@registry+https://github.com/rust-lang/crates.io-index, @@g' | sed 's/\x1b\[[0-9;]*m//g' > target/LICENSES-THIRD-PARTY-NATIVE
+# Going to exonum-java-binding-core/rust directory.
+backup_dir=${PWD}
+script_dir=$(dirname $(readlink -f "$0"))
```

Comment:
This script seems to have some required parameters, I would document them.

Reply:
Well, actually, it does not. It can be launched from any directory and this code is needed to place license file inside specific directory. After that, script will `cd` back to the directory it was called from.